guides/learning_guide.rst

```diff
@@ -0,0 +1,1094 @@
+analysis, and ML/AI techniques, the computational science community has been
+embracing much more diverse workflow solutions than ever before. These trends
+are already pushing the limits of the existing workload management products.
+Flux has been able to provide innovative solutions. Furthermore, our development
+team brought a co-design strategy to early scientific use cases, resulting in feature
+enrichments and further performance and scalability improvements. With the
+first-class workflow-enabling support of FLux, the software complexity of domain-
+specific workflow management software was greatly reduced, often to just a thin
+wrapper around Flux, and the overall end-to-end performance and scalability
+of workflows were significantly improved. This section highlights some of the
+representative scientific and engineering workflows that Flux has enabled.
+
+Cancer Research
+---------------
+
+The JDACS4C program—Joint Design of Advanced Computing Solutions for Cancer—
+is a partnership between the Department of Energy (DOE) and the National Cancer
+Institute to advance cancer research using emerging exascale HPC capabilities. The
+Pilot 2 project within JDACS4C seeks to develop effective HPC simulation methods
+to uncover the detailed characterizations of the behavior of RAS proteins on cellular
+membranes. The RAS protein family are small GTPase-signaling proteins involved
+with the control of cell division and growth. Mutations leading to increased RAS
+activity contribute to a wide range of cancers, and up to 30% of human cancers are
+linked to mutations in the RAS gene family. RAS proteins typically signal their
+downstream effectors when bound to the lipid bylayer of cellular membranes, and
+drugs that inhibit RAS activity do not exist yet.
+
+Resolving RAS membrane dynamics and aggregation is a difficult task as a micro-
+scale length and time scales are needed; yet micro-scale molecule-level details are
+also required to capture protein–protein and protein–lipid interactions. To resolve
+RAS structure and dynamics on cellular membranes, the Pilot 2 team developed the
+Multiscale Machine-Learned Modeling Infrastructure (MuMMI), which can sample
+data at the macro-scale with effective micro-scale resolution. This process is depicted
+in Figure 5.
+
+A macro model is simulated at the membrane level with 300 RAS molecules, which
+is then coupled with an ML module that drives the sampling of patches, small
+neighborhoods around an RAS molecule. These patches are then used to instantiate
+and run corresponding MD simulations.
+
+.. figure:: images/figure-5.png
+   :alt: The MuMMI Workflow
+   :align: center
+
+   Figure 5: Addressing many important biological questions requires large length- and time-scales, yet at the
+   same time molecular detail levels. Flux scalably and portably enables the MuMMI workflow to simulate
+   protein-lipid dynamics for a 1 um x 1um membrane subsection at near-atomistic resolution.
```

Comment:
space between 1 and um, and I think it's "µ" not "u"  (u00B5 - hopefully that's the right one)